src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/ContractFrame/iframe-manager.js

```diff
@@ -0,0 +1,152 @@
+!(function (e, n) {
```

Comment:
Can you a comment which explicit where this script comes from, what is its purpose and to prevent to edit it.